packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -509,6 +512,10 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
   Widget build(BuildContext context) {
     super.build(context); // See AutomaticKeepAliveClientMixin.
 
+    if (widget.direction == null) {
+      return widget.child;
```

Comment:
If `widget.child` is `null`, `performRebuild` will throw. Could you assert in the constructor that `child` and `direction` can't both be `null`?

Reply:
Ah, yeah. Good catch. Added an assertion to the constructor.